migrations/2023-05-29-094747_order-details-as-a-separate-column.sql/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE payment_intent ADD COLUMN order_details jsonb;
+
+
+ALTER TABLE payment_intent ALTER COLUMN order_details type jsonb [] using array[order_details];
```

Comment:
Two alter statements are not necessary, this can be achieved in a single command
```SQL
ALTER TABLE payment_intent ADD COLUMN order_details jsonb[];
```